ush/prune_4nco_global.sh

```diff
@@ -0,0 +1,210 @@
+        rc=$?
+        if [[ $rc -ne 0 ]]; then
+            echo "***ERROR* git checkout ${type}"
+            exit
+        fi
+    fi
+done
+
+
+# Process scripts directories and files
+cd $topdir/scripts
+rlist="exurma2p5_gsianl.sh"
+for type in $rlist; do
+    git $string ${type}*
+    rc=$?
+    if [[ $rc -ne 0 ]]; then
+        echo "***ERROR* git $string ${type}"
+        exit
+    fi
+    if [[ "$mode" = "restore" ]]; then
+        git checkout ${type}*
+        rc=$?
+        if [[ $rc -ne 0 ]]; then
+            echo "***ERROR* git checkout ${type}"
+            exit
+        fi
+    fi
+done
+
+
+# Process ush directories and files
+cd $topdir/ush
+rlist="build.comgsi EnKF com Get_Initial_Files gfs_truncate_enkf llsub para refactor_4nco_global run_arw rungsi sub"
+for type in $rlist; do
+    git $string ${type}*
+    rc=$?
+    if [[ $rc -ne 0 ]]; then
+        echo "***ERROR* git $string ${type}"
+        exit
+    fi
+    if [[ "$mode" = "restore" ]]; then
+        git checkout ${type}*
+        rc=$?
+        if [[ $rc -ne 0 ]]; then
+            echo "***ERROR* git checkout ${type}"
+            exit
+        fi
+    fi
+done
+
+
+# Process util directories and files
+cd $topdir/util
+rlist="Aero Analysis_Utilities Baseline Config Correlated_Obs DTC EFSOI FOV GEN_BE_V2.0 GMI_BUFR MODIS_AOD Misc NCEP NMC_Bkerror README Radar_Monitor Radiance_bias_correction_Utilities Radiance_Utilities Single_Observation bufr_tools global_angupdate gsienvreport.sh ndate python_utilities radar_process zero_biascoeff"
+for type in $rlist; do
+    git $string ${type}*
+    rc=$?
+    if [[ $rc -ne 0 ]]; then
+        echo "***ERROR* git $string ${type}"
+        exit
+    fi
+    if [[ "$mode" = "restore" ]]; then
+        git checkout ${type}*
+        rc=$?
+        if [[ $rc -ne 0 ]]; then
+            echo "***ERROR* git checkout ${type}"
+            exit
+        fi
+    fi
+done
+
+
+# Process util/EnKF directories and files
+cd $topdir/util/EnKF
+rlist="arw python_utilities"
+for type in $rlist; do
+    git $string ${type}*
+    rc=$?
+    if [[ $rc -ne 0 ]]; then
+        echo "***ERROR* git $string ${type}"
+        exit
+    fi
+    if [[ "$mode" = "restore" ]]; then
+        git checkout ${type}*
+        rc=$?
+        if [[ $rc -ne 0 ]]; then
+            echo "***ERROR* git checkout ${type}"
+            exit
+        fi
+    fi
+done
+
+
+# Process util/EnKF/gfs/src directories and files
+cd $topdir/util/EnKF/gfs/src
+rlist="adjustps misc preproc gribmean recenterncio_hybgain recenternemsiop_hybgain"
+for type in $rlist; do
+    git $string ${type}*
+    rc=$?
+    if [[ $rc -ne 0 ]]; then
+        echo "***ERROR* git $string ${type}"
+        exit
+    fi
+    if [[ "$mode" = "restore" ]]; then
+        git checkout ${type}*
+        rc=$?
+        if [[ $rc -ne 0 ]]; then
+            echo "***ERROR* git checkout ${type}"
+            exit
+        fi
+    fi
+done
+
+
+# Process util/Radiance_Monitor/nwprod
```

Comment:
Is this still necessary?  All old, versioned subdirectories were removed in GSI master.

Reply:
Good catch.   Versioned directories exist in release/gfsda.v16.1.0.   Since versions directories do not exist in the master, this section is not necessary.  I will remove this section from prune_4nco_global.sh